pyaedt/modules/Mesh.py

```diff
@@ -325,7 +325,7 @@ class Mesh(object):
     >>> from pyaedt import Hfss
     >>> aedtapp = Hfss()
```

Comment:
@maxcapodi78 that's small but I started to be consistent in methods docstrings. For example, rather then calling an instance of a class "app" I use the name of the class:
hfss = Hfss()
m2d = Maxwell2d()
etc.
```suggestion
    >>> hfss= Hfss()
```

and for all the docstrings below..